deployments/rialto/docker-compose.yml

```diff
@@ -144,6 +117,10 @@ services:
       - ./bridge-config:/config
     environment: &bridge-env
       RUST_LOG: runtime=trace,rpc=debug,txpool=trace
+    healthcheck:
```

Comment:
That shouldn't be necessary. The build arg should be passed when image is being built.

Reply:
This isn't part of the build: https://github.com/paritytech/parity-bridges-common/blob/master/.github/workflows/publish-docker.yml#L50

Between having it (kinda hidden) in a Github Actions workflow and here I'd prefer to have it here, but I don't have a strong opinion on this

Reply:
If it's part of the image everyone can benefit from it since it doesn't require any extra configuration.